### PR TITLE
Feature/4788 4789 4919 epic ppa

### PR DESCRIPTION
**Pull request by pascal-brause**
## Description
This epic-story branch brings up several things:
1. The new features screen for the 1.13 version with the texts for the data donation
2. The data donation screen in the onboarding
3. The data donation screen in the delta onboarding
4. The collection of the privacy preserve analytics data according to the MVP-Story (so only risk exposure and user data are collected, the other data comes with another story)
5. The storing of these data in our store
6. The submission of the analytics data to the data donations server 

The UI Tests are still in progress for the Settings, Onboarding and DeltaOnboarding.
All Tasks were already reviewed and appoved. So this not needed to be reviewed.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4788
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4919
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4789

## Screenshots

_Onboarding_
107767833-e786d680-6d35-11eb-855e-8f68b2e46fc2
107767854-ea81c700-6d35-11eb-91ae-5e4bf5faa45a PNG

_DataDonation_
107784539-dea0ff80-6d4b-11eb-857f-afb9fe698509
107784546-e06ac300-6d4b-11eb-846a-c970180849bf

_Settings_
107784473-cb8e2f80-6d4b-11eb-9852-254b463a1672
107784485-cdf08980-6d4b-11eb-8c97-128d59fe57f6

107784562-e660a400-6d4b-11eb-8ee6-71f2432bb6b2
107784570-e82a6780-6d4b-11eb-8c9b-e1aeb57b3c47

_Submission Dev Menu_
107612377-6997d200-6c46-11eb-8606-7a4f42c7d6dd
107613741-330f8680-6c49-11eb-8e45-04d63b7f63ac
107612378-6ac8ff00-6c46-11eb-9d6a-87b473a683a8
107613758-3b67c180-6c49-11eb-839a-07b989b2225d
107612385-6c92c280-6c46-11eb-8adf-14c8108506d2
107613771-44589300-6c49-11eb-930c-8206df58f498

### CODE REVIEW

**Review by naveeddotio — APPROVED** [review]

**Review by KaiTeuber — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewModels/SettingsDataDonationViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+			title: AppStrings.DataDonation.ValueSelection.Title.Age,
+			preselected: dataDonationModel.age
+		)
+		selectValueViewModel.$selectedValue .sink { [weak self] age in
+			guard self?.dataDonationModel.age != age else {
+				return
+			}
+			self?.dataDonationModel.age = age
+			self?.dataDonationModel.save()
+//			self?.reloadTableView.toggle()
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewModels/SettingsDataDonationViewModel.swift:216]: can we remove this?

Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewModels/SettingsDataDonationViewModel.swift:216]: yes, we can

Comment (marcussc) [reply src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewModels/SettingsDataDonationViewModel.swift]: image

File: src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPPAnalytics/DMPPAnalyticsViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+		case .forceSubmission:
+			return DMButtonCellViewModel(
+				text: "Force Analytics Submission",
+				textColor: .white,
+				backgroundColor: .enaColor(for: .buttonPrimary),
+				action: { [weak self] in
+					self?.submitter.forcedSubmitData(completion: { [weak self] result in
+						switch result {
+						case .success:
+							let alert = UIAlertController(title: "Data submission success", message: nil, preferredStyle: .alert)
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPPAnalytics/DMPPAnalyticsViewModel.swift:59]: Shouldn't it be a localised string?

File: src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPPAnalytics/DMPPAnalyticsViewModel.swift [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+							alert.addAction(UIAlertAction(title: "Ok", style: .cancel))
+							self?.showAlert?(alert)
+							self?.refreshTableView([TableViewSections.forceSubmission.rawValue])
+						case let .failure(error):
+							let alert = UIAlertController(title: "Data submission error", message: "Error code: \(error)", preferredStyle: .alert)
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPPAnalytics/DMPPAnalyticsViewModel.swift:64]: Shouldn't it be a localised string?

File: src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPPAnalytics/DMPPAnalyticsViewModel.swift (hunk 2 above)
Comment (marcussc) [reply src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPPAnalytics/DMPPAnalyticsViewModel.swift:59]: Its the Developer Menu, so its fine to use "fixed" strings.

File: src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPPAnalytics/DMPPAnalyticsViewModel.swift (hunk 3 above)
Comment (marcussc) [reply src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPPAnalytics/DMPPAnalyticsViewModel.swift:64]: Its the Developer Menu, so its fine to use "fixed" strings.

File: src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPPAnalytics/DMPPAnalyticsViewModel.swift (hunk 2 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPPAnalytics/DMPPAnalyticsViewModel.swift:59]: Make sense

**Pull request merged**